corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -380,3 +380,12 @@
   default: "normal"
   since: "2.48"
   force: false
+
+- name: "Enable Text To Speech"
+  description: "Enables Text To Speech feature for mobile."
```

Comment:
can we instead use the description - "Adds a text to speech button for all questions to read out the question text aloud"

Reply:
yeah that sounds much nicer, Thanks!